### DISCUSSION TITLE
Floodlight Cam Plus - Motion-Activated Lights Switch

### DISCUSSION THREAD

**Discussion by snerd9334** [tsightler/ring-mqtt#784] [Support Q&A] [answered]
I'm not able to find the Motion-Activated Lights switch as found in the Ring iOS app, under Device Settings ->Light Settings. This is a separate switch to Motion Detection. Do I need to enable an option somewhere? Thank you

<img width="414" alt="signal-2024-02-13-121919" src="https://github.com/tsightler/ring-mqtt/assets/80421289/b62caacd-1cca-436e-91d8-c86cf1b48539">

**Comment by tsightler** [comment, marked as answer]
That's because this [feature isn't supported](https://github.com/tsightler/ring-mqtt/discussions/320).  Please search for duplicate issues/discussions prior to posting.
